python/paddle/fluid/tests/unittests/test_sum_op.py

```diff
@@ -141,6 +143,64 @@ def test_w_is_selected_rows(self):
                 self.check_with_place(place, inplace)
 
 
+@unittest.skipIf(not core.supports_bfloat16(),
+                 'place does not support BF16 evaluation')
+class TestSelectedRowsSumBF16Op(TestSelectedRowsSumOp):
+    def setUp(self):
+        self.height = 10
+        self.row_numel = 12
+        self.rows = [0, 1, 2, 3, 4, 5, 6]
+        self.dtype = np.uint16
+        self.init_kernel_type()
```

Comment:
Hi, self.dtype = np.uint16
self.init_kernel_type()
but this function is not defined here so it will use parent's function which will set dtype to np.float64?

Reply:
sorry my mistake, I mistaken with another class. This class parent function is pass